lib/util.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const path = require('path');
+const { promisify } = require('util');
+const fs = require('fs');
+const readFile = promisify(fs.readFile);
+const { hasher } = require('asset-pipe-common');
+const readPkgUp = require('read-pkg-up');
+const postcss = require('postcss');
+const atImport = require('postcss-import');
+
+module.exports.identifyCssModule = function identifyCssModule (filePath) {
+    return readPkgUp({
```

Comment:
I found myself trapped between do and do not. Are we supporting latest node only?

Reply:
Well, we are in Podium. Unsure what @trygve-lie wants for the asset-pipe things, but I see no reason to not just support node 8. It will be LTS later this month anyways

Reply:
I’ll take that, run with it and not look back. 
Here comes the async/await train toot toot!